src/generated/extensions.d.ts

```diff
@@ -12,7 +12,19 @@ export interface HaveExtension {
     discoveryKey: Buffer;
     start: number;
     encodedBitfield: Buffer;
+    namespace: HaveExtension_Namespace;
 }
+export declare const HaveExtension_Namespace: {
+    readonly auth: "auth";
+    readonly config: "config";
+    readonly data: "data";
+    readonly blobIndex: "blobIndex";
+    readonly blob: "blob";
+    readonly UNRECOGNIZED: "UNRECOGNIZED";
```

Comment:
Just for my own understanding, this is added in by the ts-proto step because of the `enumsAsLiterals=true` change? isn't immediately clear to me where this comes from but could kind of guess